README.md

```diff
@@ -20,3 +20,72 @@ with a few commands. Need an ingress to serve traffic? Use juju to deploy and
 integrate a range of ingress, such as
 [traefik](https://charmhub.io/traefik-k8s), and expose your application to
 external traffic in seconds.
+
+## Contributing
```

Comment:
Should this be moved to a How-To? " How to contribute another framework", " How to contribute another integration"

Overview page should have a project overview followed by "In this documentation", "Contributing to this documentation " and "Project and Community sections". An example here: https://charmhub.io/indico

Reply:
The issue we're still working on is where to publish the documentation. We have an agreement to put the tutorial and reference in the juju SDK docs and I think those are the most critical at the moment. We'll need to keep working with the documentation team on improving this over time

Reply:
Resolving for now, please let me know if we need to discuss further 🙂